src/main/cpp/blaze_util_openbsd.cc

```diff
@@ -0,0 +1,168 @@
+// Copyright 2015 The Bazel Authors. All rights reserved.
```

Comment:
The diff seems short indeed. Could you reuse the FreeBSD file without forking it? It's a risk the two files won't stay in sync.

Reply:
The following functions are the same in `blaze_util_freebsd.cc` and `blaze_util_openbsd.cc`:

```
GetOutputRoot
WarnFilesystemType
GetMillisecondsMonotonic
SetScheduling
IsSharedLibrary
ConfigureDaemonProcess
WriteSystemSpecificProcessIdentifier
VerifyServerProcess
ExcludePathFromBackup
GetExplicitSystemLimit
```

The function bodies are 65 lines altogether.

Options include:

1. Do nothing. Keep the `blaze_util_freebsd.cc` and `blaze_util_openbsd.cc` files as they are.

2. Combine them into a single `blaze_util_bsd.cc` file that contains a bunch of `#ifdef`s. This would be messy, hard to read, and based on the flawed premise that all BSD variants will share substantially similar code.

3. Move the common functions into a `blaze_util_bsd_common.inc` file, which both `blaze_util_freebsd.cc` and `blaze_util_openbsd.cc` can `#include`. This would be more complex and indirect, and possibly not worth it for only 65 lines.

4. Something else.

How would you like to proceed? (IMO option 1 might be the least bad option.)

BTW, a few functions are also duplicated among the Linux and Darwin files.

Reply:
You're right, let's go with (1).